README.md

```diff
@@ -19,18 +19,25 @@ All contributors are welcome! If you would like to contribute:
 2. `cd rides/`
 2. Create a python virtual environment, activate it and install requirements.
   - `virtualenv rides-venv`
+  - `cd ..`
```

Comment:
Hmm, I don't think `cd ..` is what you want here.

Maybe change the `git clone` on line 18 to `git clone https://github.com/<your_username>/csh-rideboard` and `cd rides` to `cd csh-rideboard`, then you shouldn't need the `cd ..`

Reply:
Changed in commit https://github.com/ComputerScienceHouse/csh-rideboard/pull/22/commits/711d1a4ad688e98faa8a7ae76c4c812854517b00